File: airbyte-commons/src/test/java/io/airbyte/commons/lang/MoreBooleansTest.java
/*
 * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
 */

package io.airbyte.commons.lang;

import static org.junit.jupiter.api.Assertions.assertFalse;
import static org.junit.jupiter.api.Assertions.assertTrue;

import org.junit.jupiter.api.Test;

class MoreBooleansTest {

  @SuppressWarnings("ConstantConditions")
  @Test
  void evaluateNullAsFalse() {
    assertTrue(MoreBooleans.isTruthy(Boolean.TRUE));
    assertFalse(MoreBooleans.isTruthy(Boolean.FALSE));
    assertFalse(MoreBooleans.isTruthy(null));
  }

}

File: airbyte-cdk/java/airbyte-cdk/dependencies/bin/test/io/airbyte/commons/lang/MoreBooleansTest.kt
/*
 * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 */
package io.airbyte.commons.lang

import java.lang.Boolean
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

internal class MoreBooleansTest {
    @Test
    fun evaluateNullAsFalse() {
        Assertions.assertTrue(MoreBooleans.isTruthy(Boolean.TRUE))
        Assertions.assertFalse(MoreBooleans.isTruthy(Boolean.FALSE))
        Assertions.assertFalse(MoreBooleans.isTruthy(null))
    }
}
